pyne/dagmc.pyx

```diff
@@ -591,6 +599,80 @@ def get_material_set(**kw):
             mat_ids.add(d['material'])
     return mat_ids
 
+
+def get_cell_material_assignments(hdf5):
```

Comment:
Where it is seemed fine to me.

On Fri, Jan 16, 2015, 9:01 AM Kalin Kiesling <notifications@github.com>
wrote:

> In pyne/dagmc.pyx
> <https://github.com/pyne/pyne/pull/646#discussion-diff-23084546>:
>
> > @@ -591,6 +599,80 @@ def get_material_set(**kw):
> >              mat_ids.add(d['material'])
> >      return mat_ids
> >
> > +
> > +def get_cell_material_assignments(hdf5):
>
> That is much better. Is where I placed the function in the file a
> reasonable location? I wasn't sure because there seemed to be a few
> "groups" of functions in the file.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyne/pyne/pull/646/files#r23084546>.
>